src/components/ToggleSwitch/docs/index.story.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+  header,
+  api,
+  divider,
+  description,
+  importExample,
+  playground,
+  tab,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+
+import { ToggleSwitch } from '../';
+import * as Readme from '../README.md';
+import { Examples } from './examples';
+
+export default {
+  category: 'Components',
+  storyName: 'ToggleSwitch',
+  component: ToggleSwitch,
+  componentPath: '../ToggleSwitch.tsx',
+
+  componentProps: {
+    'data-hook': 'storybook-ToggleSwitch',
+    checked: true,
+  },
+  exampleProps: {},
+
+  sections: [
+    header(),
+    tabs([
+      tab({
+        title: 'Usage',
+        sections: [
+          description(Readme),
+          importExample(
+            "import { ToggleSwitch } from 'wix-ui-tpa/dist/src/components/ToggleSwitch';",
+          ),
+
+          divider(),
+
+          title('Examples'),
+
+          <Examples key="toggle-switch-examples" />,
```

Comment:
Please use the `settingsPanel` util we have, instead of this.

You can look at how the other recently created `index.story.tsx` files are structured as an example (AvatarGroup, Counter, Ratings, etc.)